sdk-jakarta/pom.xml

```diff
@@ -18,8 +18,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>a2a-java-sdk-core</artifactId>
+            <groupId>io.a2a.sdk</groupId>
```

Comment:
Any reason not to use ${project.groupId} here? Might be good to use that in case the groupId changes so then we don't need to change it in multiple places.